services/mapit/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.7'
+  
+volumes:
+  mapit-pg:
+
+x-mapit: &mapit
+  build:
+    context: .
+    dockerfile: services/mapit/Dockerfile
+  image: mapit
+  working_dir: /mapit
+
+services:
+  mapit-pg:
+    image: mdillon/postgis:9.6-alpine
+    volumes:
+      - mapit-pg:/var/lib/postgresql/data
+
+  mapit:
+    <<: *mapit
+    volumes:
+        - ~/govuk/mapit:/mapit
+    depends_on:
+      - mapit-pg
+      - nginx-proxy-app
+    working_dir: /mapit
+    environment:
+      MAPIT_DB_PASS: ''
+      DJANGO_SECRET_KEY: 'secret'
+      MAPIT_DB_HOST: 'mapit-pg'
+      PGHOST: "mapit-pg"
+      VIRTUAL_HOST: mapit.dev.gov.uk
+      HOST: 0.0.0.0
+    ports:
+      - "3108:3108"
```

Comment:
We should just have `- 3108` here, to avoid binding this port on the host system - unless we need to do that for some reason?